frontend/src/shared/helpers.tsx

```diff
@@ -1062,76 +1063,9 @@ function formatCohortRelatedDate(date: string) {
   return date?.split(" ")[0];
 }
 
-// NOTE FOR QUAN
-// TO DO - cleanup and refactor to match our existing syntax
-// spent a lot of time just making sure that the behavior matches
-// the expected behavior. the idea here is that if the input is invalid
-// that a "costCenter-validation-error" class name will be added to
-// the element's class list and this would allow the user to see if
-// their input is valid or not immediately (as they're typing it in)
-// i.e., the cell will have red text if their input is anything other than
-// a valid cost center/fund number format (^[d{5}\\/d{5}])
-function CostCenterCellEditor() {}
-
-CostCenterCellEditor.prototype.init = function (params: any) {
-  this.eGui = document.createElement("div");
-  this.eGui.innerHTML = `
-    <input value=${params.value} />
-  `;
-  this.eInput = this.eGui.querySelector("input");
-  this.eInput.addEventListener("input", this.inputChanged.bind(this));
-};
-
-CostCenterCellEditor.prototype.inputChanged = function (event: any) {
-  let val = event.target.value;
-  console.log("testing if valid cost center input = ", val);
-  if (isValidCostCenter(val)) {
-    console.log("is valid", val);
-    this.eGui.classList.remove("costCenter-validation-error");
-  } else {
-    console.log("is NOT valid", val);
-    this.eGui.classList.add("costCenter-validation-error");
-  }
-  console.log("\n\n\nupdated class list");
-  console.log(this.eGui.classList);
-};
-
-function isEmpty(val: string) {
-  return val === undefined || val === null || val === "" || val === "null";
-}
-
-CostCenterCellEditor.prototype.isCancelAfterEnd = function () {
-  let value = this.getValue();
-  if (isEmpty(value)) {
-    return true;
-  }
-  return false;
-};
-
-CostCenterCellEditor.prototype.getValue = function () {
-  return isEmpty(this.eInput.value) ? "" : this.eInput.value;
-};
-
-CostCenterCellEditor.prototype.getGui = function () {
-  return this.eGui;
-};
-
-CostCenterCellEditor.prototype.destroy = function () {
-  this.eInput.removeEventListener("input", this.inputChanged);
-};
```

Comment:
Removed a lot of this code because we aligned last Wednesday that we will perform validation after the users finish editing a cell, not during, which saves us from having to implement a custom AG Grid Cell Editor. I'll store this code in my personal notes in case we decide to change our mind in the future!